public/css/mobile.css

```diff
@@ -21,12 +21,14 @@
       transform: none;
       border-bottom-left-radius: 0px;
       border-bottom-right-radius: 0px;
-      background: #EF6262;
+      background: #F58549;
       overflow: hidden;
     }
 
     #navContainer a h3 {
-        font-size: 18px;
+        font-size: 30px;
+        padding-bottom: 10px;
+        font-weight: bold;
     }
 
     .nav-separator, .navLink, #mlh-trust-badge {
```

Comment:
Variable naming is super important for collaborative projects. If you Google "css naming conventions" tons of articles come up. While I realize we need to get this done quick, I just wanted to point out the face that the css variables do not follow a consistent format; with both camelCase and snake-case appearing in this file.